### PR TITLE
Document minimum XFS volume size

### PR DESCRIPTION
**Pull request by ejweber**
#### Which issue(s) this PR fixes:

longhorn/longhorn#8488

### CODE REVIEW

**Review by jillian-maroket — COMMENTED** [review]
@ejweber Let me know if you have concerns about the changes.

File: content/docs/1.7.0/deploy/important-notes/index.md [hunk 1]
```diff
@@ -208,6 +209,18 @@ spec:
...
 
+### Minimum XFS Filesystem Size
+
+Recent versions of `xfsprogs` (including the version used by Longhorn) [do not
+allow](https://git.kernel.org/pub/scm/fs/xfs/xfsprogs-dev.git/commit/?id=6e0ed3d19c54603f0f7d628ea04b550151d8a262) the
+creation of XFS filesystems smaller than 300 MiB. Longhorn v{{< current-version >}} will not:
+
+- Provision a volume through the CSI flow if its `resources.requests.storage < 300 Mi` and its corresponding
+  StorageClass has `fsType: xfs`, or
+- Allow a UI `Create PV/PVC` action with `File System: XFS` to complete on a volume that has `spec.size < 300 Mi`.
```
[the remark this answers is not in the document]
Comment (ejweber) [reply content/docs/1.7.0/deploy/important-notes/index.md:220]: Fixed in an amend.

File: content/docs/1.7.0/deploy/important-notes/index.md [hunk 2]
```diff
@@ -208,6 +209,18 @@ spec:
...
+
+Longhorn still allows either when cloning or restoring a volume created with a previous version.
```
[the remark this answers is not in the document]
Comment (ejweber) [reply content/docs/1.7.0/deploy/important-notes/index.md:222]: Fixed in an amend.

**Review by jillian-maroket — APPROVED** [review]
LGTM. Thanks!

**Review by derekbit — APPROVED** [review]
LGTM

**Pull request merged**